base/cvd/cuttlefish/host/commands/cvd/cli/parser/load_config.proto

```diff
@@ -86,6 +86,7 @@ message Display {
   optional uint32 height = 2;
   optional uint32 dpi = 3;
   optional uint32 refresh_rate_hertz = 4;
+  optional string overlays = 5;
```

Comment:
Use a strongly typed message here, maybe something similar to what's in launch_cvd.proto